src/components/App.js

```diff
@@ -41,6 +43,16 @@ class App extends Component {
     );
   }
 
+  componentDidMount() {
+    if (Platform.os === 'ios') {
```

Comment:
Please see the usage in bottom of this page(https://github.com/evollu/react-native-fcm).
You can understand it. :-)